deepmd/utils/data_system.py

```diff
@@ -29,7 +30,8 @@ def __init__ (self,
                   modifier = None,
                   trn_all_set = False,
                   sys_probs = None,
-                  auto_prob_style ="prob_sys_size") :
+                  auto_prob_style ="prob_sys_size",
+                  name : str = None,) :
```

Comment:
Document needs to be added.

Reply:
When doing argcheck, all fitting or descriptor have another feature 'name', and the multi-task input.json have different structure, so we add a new argcheck_mt.py

Reply:
We clarify each component in the model and combining them together in multi-task process, so the input.json file is organized by this logic

Reply:
@jxxiaoshaoye Got it. I suggest to write something like
```py
from deepmd.utils.argcheck import descrpt_local_frame_args

def descrpt_local_frame_args_mt():
    return [
        Argument("name", str, optional=True, default='', doc="Descriptor name."),
        *descrpt_local_frame_args(),
     ]
```
Besides, if every descriptor should have a name and it's not related to a specific descriptor, I think you can put it here?
https://github.com/deepmodeling/deepmd-kit/blob/3c09f770fbe97318759d807c68842755ad187e6d/deepmd/utils/argcheck_mt.py#L691
```py
    base = Argument("base", dict, [
        Argument("name", str, optional=True, default='', doc="Descriptor name."),
    ], [descrpt_variant_type_args()], doc="")
```

Reply:
If you need help, I may send a PR in one or two days.

Reply:
Ok, I think I've got your points